host_core/lib/host_core/jetstream/client.ex

```diff
@@ -0,0 +1,159 @@
+defmodule HostCore.Jetstream.Client do
+  use GenServer
+  require Logger
+
+  def start_link(config) do
+    GenServer.start_link(__MODULE__, config)
+  end
+
+  @impl true
+  def init(config) do
+    state = %{
+      lattice_prefix: config.lattice_prefix,
+      deliver_subject: config.cache_deliver_inbox
+    }
+
+    {:ok, state, {:continue, :ensure_stream}}
+  end
+
+  @impl true
+  def handle_continue(:ensure_stream, state) do
+    # TODO get rid of this
+    Process.sleep(100)
```

Comment:
Are you able to get rid of this 🙂

Reply:
Not yet. We need the other consumer servers running before we can fire up the jetstream client commands.